### PR TITLE
Fix logo alignment

### PR DESCRIPTION
**Pull request by carkod**
## Done
Fix logo alignment.

## QA
Go to /event/1/ and check that the Ubuntu summit logo is vertically aligned

### CODE REVIEW

**Comment by petesfrench** [comment]
For the error notification, you will need to find the class responsible and override it. Having a quick look I found this class:
image
It looks like it the class `.ui.form .field.error`, but check this is specific enough to override the default classes

**Comment by kewisch** [comment]
Can you also check how this looks for different themes and different size images? I seem to recall adding this as a stop gap to make logos look more centered, it wasn't easily possible due to how the header HTML is structured.

**Comment by carkod** [comment]
> For the error notification, you will need to find the class responsible and override it. Having a quick look I found this class: image It looks like it the class `.ui.form .field.error`, but check this is specific enough to override the default classes

Yeah this was what I was going to change, but it looked like it can potentially break other pages. But if you don't know anywhere else better to update this then I'll modify this one.

**Comment by carkod** [comment]
> Can you also check how this looks for different themes and different size images? I seem to recall adding this as a stop gap to make logos look more centered, it wasn't easily possible due to how the header HTML is structured.

Yeah, this is not gonna work for all images. All the styles i see here are pretty hacky. To apply it correctly to all images, we should be changing it in the theme code.

**Comment by kewisch** [comment]
We can customize templates in https://github.com/canonical/canonical-indico-customization-files, though we'd need to copy all of https://github.com/indico/indico/blob/master/indico/modules/events/templates/display/conference/base.html into a file there and then make changes to the markup. The downside is that we'd need to diff this template on each indico upgrade to make sure we're not missing any feature changes to that file.

Another - also hacky - way would be to use JS to adjust the markup, which we've done for a few select cases as well.  

Another thing we could consider is making some changes upstream that wouldn't substantially change the design but make it easier for us to adjust the theming.

File: indico_themes_canonical/static/css/_default/_forms.css [hunk 1]
```diff
@@ -40,6 +40,11 @@ input[type]:not([type="number"]):not([type="checkbox"]):not([type="radio"]):not(
   width: 100%;
 }
 
+// Override default error input bg color width default Vanilla
+.ui.form .field.error {
+  background-color: #f8e9e8;
```
Comment (kewisch) [inline indico_themes_canonical/static/css/_default/_forms.css:45]: Would recommend setting a `color` as well, we've had a few situations where not doing so bit us.

File: indico_themes_canonical/static/css/_default/_forms.css [hunk 2]
```diff
@@ -40,6 +40,11 @@ input[type]:not([type="number"]):not([type="checkbox"]):not([type="radio"]):not(
   width: 100%;
 }
 
+// Override default error input bg color width default Vanilla
```
Comment (kewisch) [inline indico_themes_canonical/static/css/_default/_forms.css:43]: /* */ to align with css spec

**Comment by kewisch** [comment]
We don't only have portrait logos, this will break landscape logos. Concluding my above comment, I'd be ok changing the template, I know getting the images right here isn't straightforward.

image

**Comment by kewisch** [comment]
If we are changing the template markup there are some considerations to make:

Each conference has a logo image you can set in the management interface. If there is a way to make an Ubuntu Summit theme with the lines in the background and just using the header image perfectly placed, this would alleviate the need for theme updates per summit. Though it might also again make it quite hard to get it placed correctly. 

An alternative might be to make the Ubuntu Summit theme a simple gradient orange and the conference image would be the full header logo with all the lines. Styles would make it overflow into the header correctly.

To make things more complicated, the image set in the management interface is unfortunately also the social image, which means we might need to make some adjustments to avoid inconvenient cropping. https://www.opengraph.xyz/url/https%3A%2F%2Fevents.canonical.com%2Fevent%2F31%2F shows that the image we're using now doesn't work very well. A better workaround for this might be to upstream a feature to set a separate social image.

**Comment by anthonydillon** [comment]
@carkod what is the status of this PR?

**Comment by carkod** [comment]
Superseded https://github.com/canonical/canonical-indico-themes/pull/40

**Pull request closed without merging**